Resources/Private/Fusion/Content/TextWithImage.fusion

```diff
@@ -0,0 +1,16 @@
+##
+# "Text" element
+#
+prototype(Neos.Demo:Content.TextWithImage) < prototype(Neos.Neos:ContentComponent) {
+    text = Neos.Neos:Editable {
+        property = 'text'
+    }
+
+    renderer = afx`
+        <div>
```

Comment:
The alignment classes are not rendered anymore, even though it's set in the backend. Example on `/en/features/text-and-images.html`.